code/game/machinery/medical_kiosk.dm

```diff
@@ -231,7 +231,7 @@
 	var/addict_list = list()
 	var/hallucination_status = "Patient is not hallucinating."
 
-	if(altPatient.reagents.reagent_list.len)	//Chemical Analysis details.
+	if(altPatient.reagents?.reagent_list.len)	//Chemical Analysis details.
```

Comment:
Should have mentioned this earlier, but you should use LAZYLEN() for your list checking needs, it's a little less hacky and doesn't require a ?.